### ISSUE TITLE
Support for Elgato EVE History

### ISSUE THREAD

**Issue by GSanCap** [hannseman/homebridge-mi-hygrothermograph#8]
Because the nature of the Temperature and Humidity data could be interesting add support for Elgato Eve History Service (https://github.com/simont77/fakegato-history) throught Fakegato History

Thanks in advance

**Comment by hannseman** [comment]
Good idea! I’ve also been interested to add an an integration with InfluxDB to view graphs with grafana.

I’ve been thinking of adding an abstract LoggingService. Implementations of this interface could be stuff like SysLogService, InfluxDbService and ElgatoEveHistoryService. 

The services would be instantiated with config in index.js and then injected into the accessory and used for adding log records.

For this to work access to the config in index.js is required and I haven’t been able to find it on the passed homebridge-object but I’ll have another look when I have time.

**Comment by hannseman** [comment]
Looks like the config can be accessed from `homebridge.user.config`.

**Comment by hannseman** [comment]
Looks like fakegato-history is very dependent on homebridge so it can’t be implemented as a LoggingService. 

The best approach is probably to make a subclass of HygrothermographAccessory which sets up fakegato. This subclass will be used instead of the default one when fakegato is configured.
